### ISSUE TITLE
Gnosis started to receive rapid and continue "Discovered new block ..." messages

### ISSUE THREAD

**Issue by tmm360** [NethermindEth/nethermind#4946]
**Describe the bug**
I've started to receive hundreds per second of messages of this type:
```
2022-11-30 18:10:51.3320|Discovered new block 25234048 17:31:15 (0xc0bc4a...d003a6), tx count: 153 miner 0x6dc0c0be4c8b2dfe750156dc7d59faabfb5b923d, sent by [Peer|eth66|25234246|      10.0.0.13:45318], with AuRa step 333965895
2022-11-30 18:10:51.3320|Discovered new block 25234049 17:31:50 (0xb59610...c01dff), tx count: 2 miner 0x35770ef700ff88d5f650597068e3aaf051f3d5a4, sent by [Peer|eth66|25234246|      10.0.0.13:45318], with AuRa step 333965902
2022-11-30 18:10:51.3320|Discovered new block 25233963 17:14:40 (0xdde80b...d95e1a), tx count: 120 miner 0x1438087186fdbfd4c256fa2df446921e30e54df8, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965696
2022-11-30 18:10:51.3320|Discovered new block 25233780 16:52:15 (0x24fa52...19ac61), tx count: 4 miner 0x657e832b1a67cdef9e117afd2f419387259fa93e, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965427
2022-11-30 18:10:51.3368|Discovered new block 25233912 17:12:50 (0x845d80...6f659a), tx count: 1 miner 0x67342ec1440bfb760ba9805a7735e0d12bb9fec7, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965674
2022-11-30 18:10:51.3368|Discovered new block 25233945 17:12:20 (0x071c62...259480), tx count: 159 miner 0x1438087186fdbfd4c256fa2df446921e30e54df8, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965668
2022-11-30 18:10:51.3397|Discovered new block 25233894 17:03:20 (0x868178...e88cb1), tx count: 1 miner 0xd653a665ff07d48fe0f239d184f28a54b00ec1ce, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965560
2022-11-30 18:10:51.3397|Discovered new block 25234026 17:26:45 (0xa62201...65dab8), tx count: 4 miner 0xb13851d5bec6c6ed331ce117ab38165e41477853, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965841
2022-11-30 18:10:51.3397|Discovered new block 25233968 17:15:35 (0xddf33a...f617e8), tx count: 1 miner 0x657e832b1a67cdef9e117afd2f419387259fa93e, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965707
2022-11-30 18:10:51.3397|Discovered new block 25234115 17:44:55 (0x2fe574...6e2ed8), tx count: 1 miner 0x1a740616e96e07d86203707c1619d9871614922a, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333966059
2022-11-30 18:10:51.3397|Discovered new block 25233920 17:07:25 (0xa9952e...9cf218), tx count: 7 miner 0x657e832b1a67cdef9e117afd2f419387259fa93e, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965609
2022-11-30 18:10:51.3397|Discovered new block 25233806 16:54:35 (0x14c0ca...f3c823), tx count: 10 miner 0x657e832b1a67cdef9e117afd2f419387259fa93e, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965455
2022-11-30 18:10:51.3455|Discovered new block 25233967 17:15:25 (0x10de76...ca8f9e), tx count: 10 miner 0xdb1c683758f493cef2e7089a3640502ab306322a, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965705
2022-11-30 18:10:51.3455|Discovered new block 25233795 16:53:35 (0x604dec...c7e4dc), tx count: 2 miner 0x1a740616e96e07d86203707c1619d9871614922a, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965443
2022-11-30 18:10:51.3455|Discovered new block 25233847 16:58:15 (0xa29cf7...52c187), tx count: 5 miner 0x1a740616e96e07d86203707c1619d9871614922a, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965499
2022-11-30 18:10:51.3455|Discovered new block 25233737 16:48:25 (0xc2b05e...e6748e), tx count: 3 miner 0x571846e42308df2dad8ed792f44a8bfddf0acb4d, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965381
2022-11-30 18:10:51.3479|Discovered new block 25233738 16:48:30 (0x066c32...a0621f), tx count: 8 miner 0x010101685223dd44a6ea8545209bcdaab157b540, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965382
2022-11-30 18:10:51.3709|Discovered new block 25234095 17:41:50 (0x855410...0c2b0c), tx count: 0 miner 0xd653a665ff07d48fe0f239d184f28a54b00ec1ce, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333966022
2022-11-30 18:10:51.3730|Discovered new block 25233754 16:49:55 (0x8684dd...90c650), tx count: 5 miner 0x657e832b1a67cdef9e117afd2f419387259fa93e, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965399
2022-11-30 18:10:51.3984|Discovered new block 25234007 17:23:00 (0x382352...92ff74), tx count: 12 miner 0x0000999dc55126ca626c20377f0045946db69b6e, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965796
2022-11-30 18:10:51.3984|Discovered new block 25233820 16:55:50 (0x92ab3a...5b5ded), tx count: 19 miner 0x10aae121b3c62f3dafec9cc46c27b4c1dfe4a835, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965470
2022-11-30 18:10:51.3984|Discovered new block 25233894 17:02:35 (0x3a72bd...d3b18b), tx count: 32 miner 0xdb1c683758f493cef2e7089a3640502ab306322a, sent by [Peer|eth66|25234246|      10.0.0.13:53654], with AuRa step 333965551
2022-11-30 18:10:51.4070|Discovered new block 25233823 16:56:10 (0x6fcb82...0211b7), tx count: 3 miner 0x0000999dc55126ca626c20377f0045946db69b6e, sent by [Peer|eth66|25234246|      10.0.0.13:45318], with AuRa step 333965474
```
My node is full synced with gnosis mainchain, running 1.14.6.
Logs always show new blocks into 25233000 and 25234999 range.

You can also see when it started, and it made my node to crush a couple of times:
![image](https://user-images.githubusercontent.com/7620704/204877518-57400c56-193d-4467-87cc-e1dc629777c4.png)

**To Reproduce**
It's a simple nethermind synced with gnosis mainnet.

**Expected behavior**
It should work as always.

**Desktop (please complete the following information):**
Running on Linux docker.

**Comment by quickchase** [comment]
I'm observing the same thing on my nodes

**Comment by quickchase** [comment]
Hotfix: https://github.com/NethermindEth/nethermind/releases/tag/1.14.7

**Comment by MarekM25** [comment]
Fixed in version 1.14.7 :)
